### PR TITLE
feat: update sleep and container name to work on both Mac and Linux

### PR DESCRIPTION
**Pull request by jcstein**
## Overview

Replaces #5 

## Problem 1

Example on macOS when container is named `nitro-testnode-da-1 `:

```bash
✔ Container nitro-testnode-da-1              Started                   0.6s
Waiting for http://localhost:26659/header/1 to come up..........................................................................Done!
Error response from daemon: No such container: nitro-testnode_da_1
```

Example on Ubuntu when container is named `nitro-testnode_da_1 `:
```bash
✔ Container nitro-testnode_da_1              Started                   0.6s
Waiting for http://localhost:26659/header/1 to come up..........................................................................Done!
Error response from daemon: No such container: nitro-testnode-da-1
```

## Solution 1

Give the container a name: `da-celestia`, keeping things standard across different OS.

```bash
 ✔ Container da-celestia                      Started                   0.1s
Waiting for http://localhost:26659/header/1 to come up...........................................................Done!
== Generating l1 keys
```
https://github.com/celestiaorg/nitro-testnode/pull/10/commits/4763e342a169ca9cff826752d3a0ef68ff67fc34

## Problem 2

Using `sleep 100s` on Mac does not run the same way as ubuntu, also, the docker file has 2 different syntax for 2 different sleep commands, with and without the `s`.

https://github.com/celestiaorg/nitro-testnode/blob/d5635fba3499adcd5713a19ecfd2c6ea19f8c0e6/test-node.bash#L262
https://github.com/celestiaorg/nitro-testnode/blob/d5635fba3499adcd5713a19ecfd2c6ea19f8c0e6/test-node.bash#L343

## Solution 2

Use standard `sleep x` from the original script, without the `s`, allowing this to run on all machines.

https://github.com/celestiaorg/nitro-testnode/pull/10/commits/1d53c496d6a2a38522ba1b1bad1f19671d39007e

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

**Comment by jcstein** [comment]
[Tested successfully, see full logs](https://app.warp.dev/block/RU5x5NAzML3DMG61o7rghu)

**Comment by jcstein** [comment]
Reminder: after this is merged, the commit of `nitro-testnode` on `nitro` will need updated

File: docker-compose.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 version: "3.9"
 services:
   da:
+    container_name: da-celestia
```
Comment (Ferret-san) [inline docker-compose.yaml:4]: what's the purpose of adding the container name (other than for labeling reasons)?

Comment (Ferret-san) [reply docker-compose.yaml:4]: ah nvm, I see why

**Review by Ferret-san — APPROVED** [review]
image

File: docker-compose.yaml (hunk 1 above)
Comment (jcstein) [reply docker-compose.yaml:4]: 🫡

**Pull request merged**